tests/function/test_cli.py

```diff
@@ -111,7 +114,7 @@ def test_interval(self, virtwho):
         result = virtwho.run_cli(oneshot=False, interval=60, wait=60)
         assert (result['send'] == 1
                 and result['interval'] == 60
-                and result['loop'] == 60)
+                and (result['loop'] == 60 or result['loop'] == 61))
```

Comment:
when debug with vcenter, the loop time always was 61, not the exact 60 because it printed much `esx.suds` logs, but I think the delay of 1s or 2s are reasonable.

Reply:
will it be shorter than 60? such as 59s, if so, we could also consider:
59 <= result['loop'] <= 61

Reply:
the loop time should not be <60 because of the least interval time is 60s.

Reply:
hm, don't know what I was thinking about